R/utilities-cmdstan.R

```diff
@@ -0,0 +1,14 @@
+# Utilities for CmdStanR
```

Comment:
None of these functions are used in the current implementation. So, this new file (`utilities-cmdstan.R`) can be removed.

Reply:
Great catch...totally agreed